README.md

```diff
@@ -36,13 +36,33 @@ We recommend using MacOS to work with the EDB Docs application.
 
 1. Install Python 3 with `brew install python3`, if it's not already installed. (Use `python3 -V` to check that you have version 3.8 or higher.) Python is not needed for the core Gatsby system, but is required by several source scripts.
 
-1. Install Yarn with `npm i -g yarn`. Yarn is the package manager we're using for this project, instead of NPM. NPM may fail with a permissions related issue. To fix that, ensure that your user account owns the required directory: `sudo chown -R $(whoami) /usr/local/lib/node_modules`
+1. NPM is the package manager we're using for this project. NPM may fail with a permissions related issue. To fix that, ensure that your user account owns the required directory: `sudo chown -R $(whoami) /usr/local/lib/node_modules`
 
 1. Install Gatsby with `npm i -g gatsby-cli`. Gatsby is the software that powers the EDB Docs site.
 
-1. Install all required packages by running `yarn`.
+1. If you're a member of the EnterpriseDB Github Org, you'll want to follow these steps so that they're pulled in the next steps.
 
-1. And finally, you can start up the site locally with `yarn develop`, which should make it live at `http://localhost:8000/`. Huzzah!
+   1. Run `brew install direnv` if you don't have it already.  Then [follow these steps to add it to your shell.](https://direnv.net/docs/hook.html).
+
+   1. Create a Github token which can pull private packages. [You can learn how to do that here.](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages)
+
+   1. Once you have that token, create a file in the root of the project called `.envrc.local` and add this code to it `export NPM_TOKEN="your-token-here"`.  That will be utilized in the NPM_TOKEN variable in the .npmrc file.
+
+1. If you're **not** a member of the EnterpriseDB Github Org, you'll have to follow these steps:
+
+  1. In the terminal, navigate to `icons-pkg/`, located folder in the root of the repository: `cd icons-pkg/`.
+
+  1. Link the icons package to you global npm packages: `npm link`.
```

Comment:
This (and the corresponding install step) strikes me as potentially error-prone - particularly for folks who have a working setup today and will need to go back & do stuff to keep it that way; can we capture this whole process in a script or something?

Reply:
@josh-heyer Jake is working on Dockerizing Docs to make things easier for the not expert users and I think that's going to help with this specifically. Stay tuned.